### ISSUE TITLE
A Question about code usage

### ISSUE THREAD

**Issue by nwhxy** [nadeemlab/DoseRTX#3]
Can this code directly output the predicted dose just from the CT image without Dose image after training？

**Comment by sanadeem** [comment]
Yes, this code will output the 3D dose distribution from just the input (CT image + OAR/PTV contours). Please take a look at Figure 1 in our published PMB'22 paper (https://arxiv.org/pdf/2207.03414.pdf) for more details.

**Comment by nwhxy** [comment]
Thank you!But I have another problem,in the RTSTRUCT of my data, there is no esophageal data, can I use other areas to replace this part for training？

**Comment by sanadeem** [comment]
Yes absolutely. You still have the CT info, even if some OAR is missing in RTSTRUCT.
